modules/ols-activating-token-quota-limits.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * TBD
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-activating-token-quota-limits_{context}"]
+= Activating token quota limits
+
+Activate token quota limits for the {ols-long} Service by defining key-value pairs in the `ConfigMap` resource. The {ols-long} pod mounts the `ConfigMap` resource as a volume, enabling access to the file stored within it. The `OLSConfig` Custom Resource (CR) references the `ConfigMap` resource to obtain the quota limit information.
```

Comment:
shouldn't we say:
Service by defining key-value pairs in the `quota-limit` instance of the `ConfigMap` resource.
?

Or can we not assured it will be named this way?
We're editing the configmap instance, instead of creating it which leads me to believe the configmap is automatically created. If so, we should know what the name is no?

Reply:
Thank you @JoaoFula. I'll confirm with @tisnik if I should use `quota-limit`. I'm not sure if it will always be referred to that way, but in this example it could be. 

You also bring up a good point about hte ConfigMap. When I wrote this, I was assuming that a ConfigMap already existed. I don't think a ConfigMap is automatically generated, but I'll also have to check with Pavel on that, too. 

@tisnik does it make sense to add a step to create the ConfigMap? I believe this is the syntax to do so: `oc create configmap <configmap-name> --from-file=<path-to-file>`

Reply:
Met with Sergey and made updates.